3.5.3/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM openjdk:8-jre-alpine
+MAINTAINER Elisey Zanko <elisey.zanko@gmail.com>
+
+
+# Install required packages
+RUN apk add --no-cache \
+    bash \
+    su-exec
+
+ENV ZOO_USER=zookeeper \
+    ZOO_CONF_DIR=/conf \
+    ZOO_DATA_DIR=/data \
+    ZOO_DATA_LOG_DIR=/datalog \
+    ZOO_PORT=2181 \
+    ZOO_TICK_TIME=2000 \
+    ZOO_INIT_LIMIT=5 \
+    ZOO_SYNC_LIMIT=2 \
+    ZOO_MAX_CLIENT_CNXNS=60
+
+# Add a user and make dirs
+RUN set -ex; \
+    adduser -D "$ZOO_USER"; \
+    mkdir -p "$ZOO_DATA_LOG_DIR" "$ZOO_DATA_DIR" "$ZOO_CONF_DIR"; \
+    chown "$ZOO_USER:$ZOO_USER" "$ZOO_DATA_LOG_DIR" "$ZOO_DATA_DIR" "$ZOO_CONF_DIR"
+
+ARG DISTRO_NAME=zookeeper-3.5.3-beta
+
+# Download Apache Zookeeper, verify its PGP signature, untar and clean up
+RUN set -ex; \
+    apk add --no-cache --virtual .build-deps \
+        gnupg; \
+    wget -q "http://www.apache.org/dist/zookeeper/$DISTRO_NAME/$DISTRO_NAME.tar.gz"; \
+    wget -q "http://www.apache.org/dist/zookeeper/$DISTRO_NAME/$DISTRO_NAME.tar.gz.asc"; \
+    wget -q "http://www.apache.org/dist/zookeeper/KEYS"; \
+    export GNUPGHOME="$(mktemp -d)"; \
+    gpg --import KEYS ; \
+    gpg --batch --verify "$DISTRO_NAME.tar.gz.asc" "$DISTRO_NAME.tar.gz"; \
+    # TODO: the distribution given by apache is not gzipped correctly - unzip correctly when fixed
+    tar -xf "$DISTRO_NAME.tar.gz"; \
+    mv "$DISTRO_NAME/conf/"* "$ZOO_CONF_DIR"; \
+    rm -rf "$GNUPGHOME" "$DISTRO_NAME.tar.gz" "$DISTRO_NAME.tar.gz.asc"; \
+    rm -rf "/$DISTRO_NAME/CHANGES.txt" \
```

Comment:
It this OK to remove the files from the official distribution? Are you sure this doesn't violate the license?